andela/travel_tool_front
# #160815252 Fix some ui issues

#### What does this PR do?
Fixes some UI issues

#### Description of Task to be completed?
- mark notification as read when  message icon is clicked
- change the colour of requestId on the request table to blue
- align the status of the request of the request on the request table to the left

#### How should this be manually tested?
- Clone the repository and change directory into the folder `cd travel_tool_front`
- Checkout from develop to this branch and pull the branch
- Start the application using `make start`
- Login to the application
- The status of a request should be properly aligned
- The color of the request id should be blue
- Click the message icon for a notification, it should mark the notification as read

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[160815252](https://www.pivotaltracker.com/story/show/160815252)

#### Screenshots (if appropriate)
![screen shot 2018-10-04 at 7 41 37 pm](https://user-images.githubusercontent.com/25783086/46495534-8e294700-c80d-11e8-8e32-7fd36102e0d3.png)


#### Questions:
None
